src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -606,6 +746,11 @@ protected virtual void OnBorderBrushChanged(Brush oldValue, Brush newValue)
 
 		protected virtual void OnFocusStateChanged(FocusState oldValue, FocusState newValue)
 		{
+			if (newValue == FocusState.Unfocused && oldValue == newValue)
```

Comment:
This is a DP callback, under what circumstances is it called with an unchanged value?

Reply:
During the initialization on Android it's happening a lot of time.